lib/sneakers/spawner.rb

```diff
@@ -14,7 +14,11 @@ def self.spawn
       worker_config = YAML.load(File.read(worker_group_config_file))
       worker_config.keys.each do |group_name|
         @pids << fork do
-          @exec_hash = {"WORKERS"=> worker_config[group_name]['classes'], "WORKER_COUNT" => worker_config[group_name]["workers"].to_s}
+          @exec_hash = {
+            "WORKERS"=> worker_config[group_name]['classes'],
+            "WORKER_COUNT" => worker_config[group_name]["workers"].to_s,
+            "PID_PATH" => worker_config[group_name]['pid_path'].to_s
```

Comment:
Hey guys - should we say that forks/signal based code should be tested with an integration test as a general principle? or maybe there's a mock library for Ruby that would help mock these kind of interactions with the OS?